react/Checkbox/Checkbox.js

```diff
@@ -22,10 +22,10 @@ export default class Checkbox extends Component {
 
   static propTypes = {
     id: PropTypes.string.isRequired,
-    label: PropTypes.string.isRequired,
```

Comment:
Ah yes that's true.  Perhaps the prop type should be one of string or element then to make it a bit stricter?